pkg/config/config.go

```diff
@@ -428,6 +429,9 @@ func initConfig(config Config) {
 	config.SetKnown("process_config.additional_endpoints.*")
 	config.SetKnown("process_config.container_source")
 	config.SetKnown("process_config.intervals.connections")
+	config.SetKnown("process_config.expvar_port")
+	config.SetKnown("process_config.bind_host")
```

Comment:
We can remove this line now that https://github.com/DataDog/datadog-agent/pull/3739 is merged